src/pages/CodePage.tsx

```diff
@@ -0,0 +1,74 @@
+import {Link} from "react-router-dom";
+import "../components/modal/modal.scss";
+import Button from "../components/Button";
+import Code from "../components/Code";
+import {Formik, Field, Form} from "formik";
+import {CODE} from "../constants";
+
+interface FormErrors {
+  [key: string]: string
+}
+
+interface FormValue {
+  code: string;
+}
+
+const CodeRegex = new RegExp('[0-9a-z]{5}', 'i');
+
+const validate = async (values: FormValue) => {
+  let errors: FormErrors = {};
+  if (!CodeRegex.test(values.code)) {
```

Comment:
'12345-!!!!' will pass the chek here without error - is it ok?

Reply:
I can't enter '12345-!!!!', only 5 letters and numbers together.

Reply:
Ok, resolve this comment